cime_config/allactive/config_compsets.xml

```diff
@@ -384,40 +296,13 @@
   <lname>SSP585SOI_EAM%CMIP6_ELM%CNPECACNTBC_MPASSI_MPASO_MOSART_SGLC_SWAV_BGC%BDRD</lname>
 </compset>
 
-<!-- E3SM backup compsets -->
-<compset>
- <alias>A_WCYCL1850_v0atm</alias> 
- <lname>1850_EAM_ELM%SP_MPASSI_MPASO_MOSART_SGLC_SWAV</lname>
-</compset>
-
-<compset>
- <alias>A_WCYCL2000_v0atm</alias>
- <lname>2000_EAM_ELM%SP_MPASSI_MPASO_MOSART_SGLC_SWAVi</lname>
-</compset>
-
-<compset>
- <alias>A_WCYCL1850S_v0atm</alias> 
- <lname>1850SOI_EAM_ELM%SP_MPASSI_MPASO_MOSART_SGLC_SWAV</lname>
-</compset>
-
-<compset>
- <alias>A_WCYCL2000S_v0atm</alias>
- <lname>2000SOI_EAM_ELM%SP_MPASSI_MPASO_MOSART_SGLC_SWAVi</lname>
-</compset>
-
-
 <!-- E3SM BG compsets -->
 
 <compset>
   <alias>A_BG1850CN</alias>
   <lname>1850_EAM_ELM%CN_MPASSI_MPASO_MOSART_MALI%SIA_SWAV</lname>
 </compset>
 
-<compset>
-  <alias>A_BGWCYCL1850</alias>
-  <lname>1850_EAM%AV1C-L_ELM%SPBC_MPASSI_MPASO_MOSART_MALI%STATIC_SWAV</lname>
-</compset>
-
```

Comment:
We may want to replace this one with something else

Reply:
How about making it BGWCYCL1850_CMIP6 and changing "EAM%AV1C" to "EAM%CMIP6"?

Reply:
A_BGWCYCL1850 has been removed. BGC compsets for v2 are yet to be finalized.